commerce-test-util/src/main/java/com/liferay/commerce/test/util/TestCommerceContext.java

```diff
@@ -95,5 +95,4 @@ public long getSiteGroupId() throws PortalException {
 	private final CommerceOrder _commerceOrder;
 	private final Group _contextGroup;
 	private final User _contextUser;
-
```

Comment:
this empty line is required -- all classes should have an empty line before the closing brace.